### PR TITLE
WIP: [CFM-1274] Project navigation rearrange fixes

### PR DESCRIPTION
**Pull request by ordavidil**
https://issuetracker.amplexor.com/jira/browse/CFM-1274

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_group_menu/c4m_og_group_menu.module [hunk 1]
```diff
@@ -109,7 +109,7 @@ function c4m_og_group_menu_form_og_menu_edit_item_form_alter(&$form, &$form_stat
 
   // The select element of the wiki pages should not be displayed on a default
   // menu link.
-  if ($form['link_path']['#value'] != 'wiki') {
+  if ($form['link_path']['#type'] == 'value') {
```
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_group_menu/c4m_og_group_menu.module]: Don't think so.
For any default link, there should be no option to link it to wiki page path.
For example, link of News item should point to /news and not any other path.

**Comment by anvmn** [comment]
Closing in favor of #1553

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_group_menu/c4m_og_group_menu.module (hunk 1 above)
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_group_menu/c4m_og_group_menu.module]: This alter is adding the wiki select, why do we need it on the news item?
I don't understand your comment ...

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_group_menu/c4m_og_group_menu.module]: This line is removing the wiki select

**Pull request closed without merging**